boards/xtensa/heltec_wifi_lora32_v2/heltec_wifi_lora32_v2-pinctrl.dtsi

```diff
@@ -20,6 +20,17 @@
 		};
 	};
 
+	spim3_default: spim3_default {
+		group1 {
+			pinmux = <SPIM3_MISO_GPIO19>,
+				 <SPIM3_SCLK_GPIO5>;
```

Comment:
I wanted to stay consistent across other ESP32 boards by defining the `pinmux` node the same way, i.e. including the `CS#` PIN.

But adding the `SPIM3_CSEL_GPIO18` to the group breaks the SPI communication and I do not know why.

Maybe someone can explain me what is wrong?

However, I guess it makes sens to leave it undefined. It is setup via the `cs-gpio` from the `spi3` node.

Also, all the PINs are connected directly from the ESP32 to the SX1276 without any pull-up or pull-down (according to the schematics).

Can someone explain to me if I should add `output-low` or up or `bias-pull-down` or up?

Thanks

Reply:
I would be pleased if someone can give some tips about what I mentioned above to make sure this is the right solution. Thanks.

Reply:
@gportay Hi, 
`However, I guess it makes sens to leave it undefined. It is setup via the cs-gpio from the spi3 node.`
Regarding the CS pin, you are correct, it is already there in `cs-gpio`.  

You can keep `output-low;` in the MOSI pins as well. It sets this pin as output with low level.  

About the gpio/dio connected between ESP32 and SX1762, the SX1762 datasheet does not mention that a pull-up is required in those pins, so it seems you are good as is.